website/docs/guides/orchestration/set-up-ci/2-quick-setup.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Quickstart: Enable CI in 15 minutes"
+slug: in-15-minutes
+description: Find issues before they are deployed to production with dbt Cloud's Slim CI.
+---
+
+## Prerequisites
+
+As part of your initial dbt Cloud setup, you should already have Development and Production environments configured. Let's recap what each does:
+
+- Your **Development environment** powers the IDE. Each user has individual credentials, and builds into an individual dev schema. Nothing you do here impacts any of your colleagues.
+- Your **Production environment** brings the canonical version of your project to life for downstream consumers. There is a single set of deployment credentials, and everything is built into your production schema(s).
+
+In this guide, we're going to add a **Staging environment**, where proposed changes can be validated in the context of the entire project without impacting production systems. We will use a single set of deployment credentials (like the Prod environment), but staging models are built in a separate location to avoid impacting others (like the Dev environment).
```

Comment:
'staging models' here is confusing, that means something else elsewhere

Reply:
Good shout - it's now a CI environment and I've avoided calling them staging models anywhere